### PR TITLE
Slow loading of the OpenShift explorer caused by duplicate oc commands #4289

### PR DESCRIPTION
**Pull request by vrubezhny**
Fixes: #4289

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -52,13 +55,24 @@ export class CliChannel {
...
 
-    async executeTool(command: CommandText, opts?: cp.ExecOptions, fail = true): Promise<CliExitData> {
+    async executeTool(command: CommandText, opts?: cp.ExecOptions, fail = true, executionContext?: ExecutionContext): Promise<CliExitData> {
         const commandActual = command.toString();
         const commandPrivacy = command.privacyMode(true).toString();
         const [cmd] = commandActual.split(' ');
         const toolLocation = await ToolsConfig.detect(cmd);
         const optsCopy = CliChannel.applyEnv(opts, CliChannel.createTelemetryEnv())
+
+        const key = `${command.command} ${[command.parameter, ...command.options.map((o)=>o.toString())]}`;
```
Comment (datho7561) [inline src/cli.ts:65]: I believe that `commandActual` is the same string as this `key`, and that has already been computed

**Review by datho7561 — CHANGES_REQUESTED** [review]
Small fix, but otherwise is working amazingly!

**Review by datho7561 — DISMISSED** [review]
Seems good. Thanks, Victor!

File: src/cli.ts (hunk 1 above)
Comment (vrubezhny) [reply src/cli.ts]: It's not the same, but actually using `commandActual` value is better.
Fixed

**Review by datho7561 — APPROVED** [review]

**Pull request merged**